rgb-contract-implementation/schema/supported-schemas.md

```diff
@@ -0,0 +1,132 @@
+# List of supported Schemas
+
+This section lists the schemas that have been officially implemented so far, describing the data
+they work with and the list of supported operations.
+
+## NIA (Non Inflatable Asset)
+
+This schema defines a non-inflatable fungible asset, involving the following data:
+- Ticker: short identifier of the asset, to be displayed on wallets and exchanges (e.g.: BTC, USDT, ...). Note
+    that there is no guarantee of uniqueness
+- Name: extended name of the asset (e.g.: Bitcoin, Tether, ...)
+- Details: optional string describing the asset in more detail
+- Precision: Number of decimal places of the standard representation of the asset (e.g.
+    would be 8 for BTC and 2 for USD)
+- Contract Terms: contains a string describing the contract and an optional media
+    attachment
+- Issued Supply: total amount of tokens created at issuance. Since the asset is not
+    inflatable, it also serves as an upper bound on the circulating supply
+
+_Note:_ Ticker, Name, Details and Precision are represented by a single object called _AssetSpec_
```

Comment:
I'd use `AssetSpec` as the first entry directly, with the 4 sub-entries, instead of mentioning it in a note